### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by LukasBoll**
Setup for tests
#8

### CODE REVIEW

**Comment by LukasBoll** [comment]
This is the current setup for the tests. We thought it would be good to get your feedback before writing more Tests.

File: package.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
     "watch": "tsc -watch -p ./",
     "pretest": "npm run compile && npm run lint",
     "lint": "eslint src --ext ts",
-    "test": "node ./out/test/runTest.js"
+    "test": "cp -R test/tutorial/. test/runTests && node ./out/test/runTest.js",
+    "testWindows": "xcopy /s .\\test\\tutorial .\\test\\runTests\\ && node ./out/test/runTest.js"
```
Comment (eneufeld) [inline package.json]: Could you check whether there is a node util or something to copy files, those windows / linux path are usually quite a pain to maintain.

**Review by eneufeld — APPROVED** [review]
Tests and setup looks good to me. 
Looking forward to more tests ;-) .

File: package.json (hunk 1 above)
Comment (LukasBoll) [reply package.json]: I did find a node module to resolve this, thanks!

**Comment by LukasBoll** [comment]
tests for extension, test for react app, closes eclipsesource#8

**Comment by LukasBoll** [comment]
We included more tests for the extension as well as the react App. The Pull request is ready to get reviewed and merged now. Thanks for your review.

**Pull request merged**